### ISSUE TITLE
Ubuntu 20.04  CUDA 11.0 tfjs-node-gpu 2.0.1 Could not load dynamic library Error, Skipping registering GPU devices

### ISSUE THREAD

**Issue by proj56578** [tensorflow/tfjs#3727]
- [ ] To get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

#### TensorFlow.js version
2.0.1
#### Describe the problem or feature request

i installed CUDA 11 on Ubuntu 20 using these 
apt clean; apt update; apt purge cuda; apt purge nvidia-*; apt autoremove; apt install cuda
install using this
https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&target_distro=Ubuntu&target_version=2004&target_type=deblocal
run post install - 
https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html#post-installation-actions

Trying to run a sample using GPU getting this error : node index.js

Overriding the gradient for 'Max'
Overriding the gradient for 'OneHot'
Overriding the gradient for 'PadV2'
Overriding the gradient for 'SpaceToBatchND'
Overriding the gradient for 'SplitV'
node-pre-gyp info This Node instance does not support builds for N-API version 5
node-pre-gyp info This Node instance does not support builds for N-API version 6
node-pre-gyp info This Node instance does not support builds for N-API version 5
node-pre-gyp info This Node instance does not support builds for N-API version 6
2020-08-03 21:38:11.446282: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2020-08-03 21:38:11.486608: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 3492985000 Hz
2020-08-03 21:38:11.487958: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x40053a0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-08-03 21:38:11.487999: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-08-03 21:38:11.488794: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcuda.so.1
2020-08-03 21:38:11.505048: I tensorflow/stream_executor/cuda/cuda_gpu_executor.cc:983] successful NUMA node read from SysFS had negative value (-1), but there must be at least one NUMA node, so returning NUMA node zero
2020-08-03 21:38:11.505923: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1618] Found device 0 with properties: 
name: TITAN RTX major: 7 minor: 5 memoryClockRate(GHz): 1.77
pciBusID: 0000:42:00.0
2020-08-03 21:38:11.505985: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcudart.so.10.0'; dlerror: libcudart.so.10.0: cannot open shared object file: No such file or directory
2020-08-03 21:38:11.506014: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcublas.so.10.0'; dlerror: libcublas.so.10.0: cannot open shared object file: No such file or directory
2020-08-03 21:38:11.506040: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcufft.so.10.0'; dlerror: libcufft.so.10.0: cannot open shared object file: No such file or directory
2020-08-03 21:38:11.506065: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcurand.so.10.0'; dlerror: libcurand.so.10.0: cannot open shared object file: No such file or directory
2020-08-03 21:38:11.506090: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcusolver.so.10.0'; dlerror: libcusolver.so.10.0: cannot open shared object file: No such file or directory
2020-08-03 21:38:11.506114: W tensorflow/stream_executor/platform/default/dso_loader.cc:55] Could not load dynamic library 'libcusparse.so.10.0'; dlerror: libcusparse.so.10.0: cannot open shared object file: No such file or directory
2020-08-03 21:38:11.508703: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library libcudnn.so.7
2020-08-03 21:38:11.508718: W tensorflow/core/common_runtime/gpu/gpu_device.cc:1641] Cannot dlopen some GPU libraries. Please make sure the missing libraries mentioned above are installed properly if you would like to use GPU. Follow the guide at https://www.tensorflow.org/install/gpu for how to download and setup the required libraries for your platform.
Skipping registering GPU devices...
2020-08-03 21:38:11.587020: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1159] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-08-03 21:38:11.587051: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1165]      0 
2020-08-03 21:38:11.587056: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1178] 0:   N 

#### Code to reproduce the bug / link to feature request

$ nvcc --version 
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2020 NVIDIA Corporation
Built on Thu_Jun_11_22:26:38_PDT_2020
Cuda compilation tools, release 11.0, V11.0.194
Build cuda_11.0_bu.TC445_37.28540450_0

$ cat /etc/*-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.1 LTS"
NAME="Ubuntu"
VERSION="20.04.1 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.1 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal

$ node -v
v11.15.0

$more index.js

// const tf = require('@tensorflow/tfjs');
// const tf = require('@tensorflow/tfjs-node');

// Optional Load the binding:
// Use '@tensorflow/tfjs-node-gpu' if running with GPU.
const tf = require('@tensorflow/tfjs-node-gpu');

// Train a simple model:
const model = tf.sequential();
model.add(tf.layers.dense({ units: 100, activation: 'relu', inputShape: [10] }));
model.add(tf.layers.dense({ units: 1, activation: 'linear' }));
model.compile({ optimizer: 'sgd', loss: 'meanSquaredError' });

const xs = tf.randomNormal([100, 10]);
const ys = tf.randomNormal([100, 1]);

model.fit(xs, ys, {
  epochs: 100,
  callbacks: {
    onEpochEnd: (epoch, log) => console.log(`Epoch ${epoch}: loss = ${log.loss}`)
  }
});

If you would like to get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

GitHub issues for this repository are tracked in the [tfjs union repository](https://github.com/tensorflow/tfjs/issues).

Please file your issue there, following the guidance in [that issue template](https://github.com/tensorflow/tfjs/blob/master/ISSUE_TEMPLATE.md).

**Comment by rthadur** [comment]
@proj56578 can you please check for a similar issue [here](https://github.com/tensorflow/tfjs/issues/2370)

**Comment by proj56578** [comment]
i think the issue might also be tfjs doesn’t support cuda 11 ?
let me try with node 10

**Comment by rthadur** [comment]
yes ,please try with 10.

**Comment by proj56578** [comment]
any idea when cuda 11 will be supported ?
i read on one of the threads here , in 2.4 ?
if that’s true , can you pls share a timeframe ?

**Comment by rthadur** [comment]
@proj56578 it depends on tensorflow core, right now latest version supported in tf core is 10.1 , thank you

**Comment by jackz314** [comment]
TensorFlow core has supported CUDA 11 for a while now, which I verified by testing out the Python version. However, I'm still having the same problem (Could not load dynamic library 'libcuxxx.so.10.0') with tfjs. Is this still not supported?

**Comment by digitalemerge** [comment]
> TensorFlow core has supported CUDA 11 for a while now, which I verified by testing out the Python version. However, I'm still having the same problem (Could not load dynamic library 'libcuxxx.so.10.0') with tfjs. Is this still not supported?

Is there any news on this regard? Still no support for cuda11
